docs/data/material/getting-started/templates/checkout/PaymentForm.tsx

```diff
@@ -1,65 +1,250 @@
+                paymentType === 'bankTransfer' ? 'primary.main' : 'divider',
+              backgroundColor:
+                paymentType === 'bankTransfer' ? 'background.default' : '',
+            }}
+          >
+            <CardActionArea onClick={() => setPaymentType('bankTransfer')}>
+              <CardContent sx={{ display: 'flex', alignItems: 'center' }}>
+                <AccountBalanceRoundedIcon color="action" sx={{ marginRight: 2 }} />
+                <Typography variant="subtitle1">US bank account</Typography>
+              </CardContent>
+            </CardActionArea>
+          </Card>
+        </RadioGroup>
+      </FormControl>
+      {paymentType === 'creditCard' && (
+        <Box
+          sx={{
+            display: 'flex',
+            justifyContent: 'center',
+          }}
+        >
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              justifyContent: 'space-between',
+              p: 3,
+              height: { xs: '300px', sm: '350px' },
+              width: { xs: '450px', sm: '525px' },
+              borderRadius: '20px',
+              border: '2px solid ',
+              borderColor: 'divider',
+              backgroundColor: 'background.paper',
+              boxShadow: '0px 4px 8px rgba(0, 0, 0, 0.05)',
+            }}
+          >
+            <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
+              <Typography variant="subtitle2">Credit card</Typography>
+              <CreditCardRoundedIcon sx={{ color: 'text.secondary' }} />
+            </Box>
+            <SimCardRoundedIcon
+              sx={{
+                fontSize: { xs: '48px', sm: '56px' },
+                transform: 'rotate(90deg)',
+                color: 'text.secondary',
+              }}
+            />
+            <Box
+              sx={{
+                display: 'flex',
+                justifyContent: 'space-between',
+                width: '100%',
+                gap: 3,
+              }}
+            >
+              <FormGrid sx={{ flexGrow: 1 }}>
+                <FormLabel required>Card number</FormLabel>
+                <OutlinedInput
+                  id="standard-basic"
+                  autoComplete="card-number"
+                  placeholder="0000 0000 0000 0000"
+                  required
+                  value={cardNumber}
+                  onChange={handleCardNumberChange}
+                />
+              </FormGrid>
+              <FormGrid sx={{ maxWidth: '20%' }}>
+                <FormLabel required>CVV</FormLabel>
+                <OutlinedInput
+                  id="cvv"
+                  autoComplete="CVV"
+                  placeholder="123"
+                  required
+                  value={cvv}
+                  onChange={handleCvvChange}
+                />
+              </FormGrid>
+            </Box>
+            <Box sx={{ display: 'flex', justifyContent: 'space-between', gap: 3 }}>
```

Comment:
```suggestion
            <Box sx={{ display: 'flex', gap: 2 }}>
```

No need for the justify-content and reducing the gap a bit